lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -755,7 +754,7 @@ static FuncDecl *deriveEncodable_encode(TypeChecker &tc, Decl *parentDecl,
                                       TypeLoc::withoutLoc(returnType),
                                       target);
   encodeDecl->setImplicit();
-  encodeDecl->setBodySynthesizer(deriveBodyEncodable_encode);
+  encodeDecl->setBody(deriveBodyEncodable_encode(encodeDecl));
```

Comment:
I think we need to continue doing this lazily. This creates extra work when just trying to use a type as Codable from another file. I think you can use `NominalTypeDecl::lookupConformance` instead.